test/DriverTest.php

```diff
@@ -349,4 +349,13 @@ private function getPermissionsFromStat(array $stat): string
     {
         return \substr(\decoct($stat["mode"]), 1);
     }
+
+    public function testChown(): \Generator
+    {
+        $fixtureDir = Fixture::path();
+
+        $original = "{$fixtureDir}/small.txt";
+        $user = fileowner($original);
+        $this->assertTrue(yield File\chown($original, $user));
+    }
```

Comment:
The new test fails for `uv`, could you have a look?

Reply:
Okay, I'll check tomorrow. Could you have a look at [this](https://github.com/amphp/sync/pull/10) please? I'd like to finish my work on the AsyncFileMutex tomorrow too.

Reply:
@kelunik I tried but it's passing for me locally. Most likely because I'm using an older ext-uv. But I do not wish to install ext-uv 0.3 on localhost yet.